src/SharedType.elm

```diff
@@ -10,3 +10,5 @@ type RepeatType
 type Tabs
     = Kodi
     | Local
+    | Audiot
```

Comment:
This should be it's own type, maybe even as subtype of the Kody type? At least that's how I think of the state

Reply:
```

type Tabs
    = Kodi KodiTabs
    | Local


type KodiTabs
    = Audio
    | Video
```

Reply:
Hi @razzeee,
I have made the required change and created the subtype of Kodi type - 
```
type KodiTabs
    = AudTab
    | VidTab
```
I have named the subtype AudTab and VidTab because the Audio & Video Type has already been defined in WSDecoder.Elm file, which caused a conflict.